docker/test/integration/features/kafka.feature

```diff
@@ -53,7 +53,7 @@ Feature: Sending data to using Kafka streaming platform using PublishKafka
     And the "success" relationship of the GetFile processor is connected to the PublishKafka
     And the "success" relationship of the GetFile processor is connected to the PutFile
 
-    And a kafka broker "broker" is set up in correspondence with the PublishKafka
+    And a kafka broker is set up in correspondence with the PublishKafka
```

Comment:
I think the second success relationship is supposed to be between PublishKafka and PutFile in the step above this.

Reply:
Awesome catch, thanks! This test wasn't even working properly having wrong properties and succeeding without using the `PublishKafka` processors. Fixed it in 80822158e9be7be0c284436fed00b70b37f0ed54